### PR TITLE
Directory in the output filename + parent_id + time spent

### PR DESCRIPTION
**Pull request by barbaragribeiro**
1. If the output filename contains a directory, it would give an error. Now it creates the directory and works;
2. Added the `parent_id` of each comment. This is important to explicit the comment threads;
3. I think it's nice to have the time spent downloading the comments, so it is printed at the end too.

### CODE REVIEW

File: downloader.py [hunk 1]
```diff
@@ -82,7 +82,9 @@ def download_comments_new_api(youtube_id, sleep=1):
                          for ncd in search_dict(response, 'nextContinuationData')] + continuations
 
         for comment in search_dict(response, 'commentRenderer'):
-            yield {'cid': comment['commentId'],
+            cid = comment['commentId']
+            yield {'cid': cid,
+                   'parent_id': cid.split('.')[0] if '.' in cid else None,
```
Comment (egbertbouman) [inline downloader.py:87]: I don't really like storing fields that can be derived from other fields. This is best done during post-processing.

File: downloader.py [hunk 2]
```diff
@@ -225,11 +229,17 @@ def main(argv):
             parser.print_usage()
             raise ValueError('you need to specify a Youtube ID and an output filename')
 
+        if '/' in output:
```
Comment (egbertbouman) [inline downloader.py:232]: To make it platform-independent, please use `os.sep`.

File: downloader.py [hunk 3]
```diff
@@ -225,11 +229,17 @@ def main(argv):
...
+            outdir = output.rsplit('/', maxsplit=1)[0]
```
Comment (egbertbouman) [inline downloader.py:233]: You can use `os.path.dirname` to make it platform-independent.

**Comment by barbaragribeiro** [comment]
> I have a few comments. Most notably, I prefer to remove `parent_id`. If you could address these issues, we can merge this PR.

Ok, I removed `parent_id` and now the directory creation is platform-independent as you pointed out ;)

**Comment by egbertbouman** [comment]
Great, thanks @barbaragribeiro!

**Pull request merged**